fixtures/arri/l10-c.json

```diff
@@ -0,0 +1,393 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "L10-C",
+  "categories": ["Dimmer", "Color Changer"],
+  "meta": {
+    "authors": ["Dyami Caliri"],
+    "createDate": "2018-10-17",
+    "lastModifyDate": "2018-10-17"
+  },
+  "links": {
+    "manual": [
+      "https://www.arri.com/resource/blob/31168/649ca2ae2935bc75686e42fbe388e07f/l5-0006896-arri-l-series-l10-user-manual-en-nov2018-l03301-data.pdf"
+    ],
+    "productPage": [
+      "https://www.arri.com/en/lighting/led/l-series-c/l10-c"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=P5aUA5M-CXU"
```

Comment:
This video is specifically about the L5-C. Please add a video that is more specific about the L10-C instead (there seem to be many: https://www.youtube.com/results?search_query=arri+l10-c).

[This video](https://www.youtube.com/watch?v=FFqDdU5SHOs) is about the whole L-Series, so it should be added to all 3 fixtures as a second video.